capa/render/default.py

```diff
@@ -19,6 +19,9 @@
 
 tabulate.PRESERVE_WHITESPACE = True
 
+MIN_LIBFUNCS_RATIO = 0.4
+MIN_API_CALLS = 10
```

Comment:
where did these numbers come from? and how should i interpret them?

Reply:
1. `MIN_LIBFUNCS_RATIO`: When the total count of library function present in a sample is less then 40%, we inform users that capa might pick false positive matches from other functions that would have been classified as library functions. I don't have any statistical data to back this up other than this hex-rays [blogpost](https://hex-rays.com/products/ida/tech/flirt/in_depth/#:~:text=Of%20course%2C%20this,the%20target%0A%20%20program.).
2. `MIN_API_CALLS`: When the sample has very few API calls, it is a strong indication that it might be packed/encrypted as regular programs tends to make a lot more than 10 (though, we have to run a benchmark across multiple sample to decide what's a good number here). For example this packed [capa-testfile](https://github.com/mandiant/capa-testfiles/blob/master/Practical%20Malware%20Analysis%20Lab%2001-02.exe_) emits 0 API features, luckily we detect that it is packed with UPX. If that weren't the case, this banner could serve as an indication that the sample might packed.

Reply:
good explanations!

would you include the key parts here as a comment?

Reply:
also i'm interested to see how frequently this message is shown to users. I don't think our dogs will identify 40% of functions in most binaries, so i'm a little concerned this message will be shown too often. 

have you had a chance to collect these stats against a large number of samples?

Reply:
I think it's still helpful information since we know there's most likely more library code than we've identified.